### DISCUSSION TITLE
Welcome to kubeedge Discussions!

### DISCUSSION THREAD

**Discussion by kevin-wangzefeng** [kubeedge/kubeedge#2813] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bensch98** [comment]
Hi all,
I am currently working on a project where I need to continuously deploy ML models and collect data from multiple Edge devices. Since my development environment itself is Ubuntu 20.04 and the edge device is a Raspberry Pi 4, everything fits so far. However, in production there will be edge devices running Windows. Is it possible to connect Edge devices running on Windows to a master node running on Ubuntu?

**Reply by kevin-wangzefeng** [reply]
> Hi all,
> I am currently working on a project where I need to continuously deploy ML models and collect data from multiple Edge devices. Since my development environment itself is Ubuntu 20.04 and the edge device is a Raspberry Pi 4, everything fits so far. However, in production there will be edge devices running Windows. Is it possible to connect Edge devices running on Windows to a master node running on Ubuntu?

Hi @bensch98, the KubeEdge components currently rely on linux, so even for cloudcore, you would need linux environment to run it.

**Comment by vickiyan** [comment]
Hi,
I'm wondering what's the mechanism for the pod on the edge to get the CRD information? e.g. the Calico CNI pod running on the edge node?  Query " sqlite3 /var/lib/kubeedge/edgecore.db" , there is no CRD metadata information. 
I need to specify "KUBERNETES_SERVICE_PORT" and "KUBERNETES_SERVICE-HOST" env in Calico CNI daemon YAML file in order to run Calico CNI pod successfully. I'm using kubeedge v.1.15 version. I'm wondering what's the mechanism behind of this? Pod should get those CRD related information relayed by Edged similar like kubelet components?  That is, edged->edgehub->cloudhub->apiserver? 
-----------------------------------------------------------------------------------
sqlite> select distinct type from meta;
listener
configmap
secret
node
pods
service
podlist
endpoints
podstatus
pod
nodestatus
-----------------------------------------
Thanks!
